js/base.js

```diff
@@ -7,6 +7,7 @@ import { OrderAPI } from "./order";
 import { ProductCollectionAPI } from "./product-collection";
 import { ProductDropAPI } from "./product-drop";
 import { ProductDropFeedbackAPI } from "./product-drop-feedback";
+import { PusherTriggerAPI } from "./pusher-trigger";

```

Comment:
Abstract the way the fact that the implementation is using Pusher. We also don't name database methods "mongoUpdateProduct" :D